lib/iris/fileformats/pp_save_rules.py

```diff
@@ -860,13 +860,18 @@ def _all_other_rules(cube, pp):
     if check_items in CF_TO_LBFC:
         pp.lbfc = CF_TO_LBFC[check_items]
 
-    # Set STASH code.
+    # Set field code.
     if (
         "STASH" in cube.attributes
         and str(cube.attributes["STASH"]) in STASH_TRANS
     ):
         pp.lbfc = STASH_TRANS[str(cube.attributes["STASH"])].field_code
 
+    # Set ensemble member number.
+    real_coord = scalar_coord(cube, "realization")
+    if real_coord is not None:
+        pp.lbrsvd[3] = real_coord.points[0]
```

Comment:
Makes sense based on the spec, and also in terms of reverse engineering of what happens during loading 👍